### PR TITLE
Modify signum

### PR DESCRIPTION
**Pull request by xiaobiaozhao**

### CODE REVIEW

File: resty_signal.c [hunk 1]
```diff
@@ -118,6 +121,28 @@ resty_signal_signum(int num)
...
+#ifdef __APPLE__
+    case RS_EMT:
+        return SIGEMT;
+
+    case RS_SYS:
+        return SIGSYS;
+
+    case RS_INFO:
+        return SIGINFO;
+#endif
```
Comment (agentzh) [inline resty_signal.c:145]: Needs a blank line here.

File: t/signum.t [hunk 2]
```diff
@@ -25,6 +26,82 @@ __DATA__
...
+                URG = 23,
+                XCPU = 24,
+                XFSZ = 25,
+                VTALRM = 26,
+                PROF = 27,
+                WINCH = 28,
+                IO = 29,
+                PWR = 30
+            } 
+            local macosx_signals = {
```
Comment (agentzh) [inline t/signum.t:61]: Needs a blank line before this line.

File: t/signum.t [hunk 3]
```diff
@@ -25,6 +26,82 @@ __DATA__
...
+            if ffi.os == "Linux" then
+                for signame, num in pairs(linux_signals) do
+                    assert(num == tonumber(signum(signame)))
+                end
+            elseif ffi.os == "OSX" then
+                for signame, num in pairs(macosx_signals) do
+                    assert(num == tonumber(signum(signame)))
+                end
+            end
+
```
Comment (agentzh) [inline t/signum.t:104]: Should remove this blank line.

**Review by agentzh — CHANGES_REQUESTED** [review]

File: resty_signal.c (hunk 1 above)
Comment (xiaobiaozhao) [reply resty_signal.c:147]: done

File: t/signum.t (hunk 2 above)
Comment (xiaobiaozhao) [reply t/signum.t:62]: done

File: t/signum.t (hunk 3 above)
Comment (xiaobiaozhao) [reply t/signum.t]: done

File: resty_signal.c [hunk 4]
```diff
@@ -24,14 +24,17 @@ enum {
...
-    RS_URG= 23,
+    RS_URG = 23,
     RS_XCPU = 24,
     RS_XFSZ = 25,
     RS_VTALRM = 26,
     RS_PROF = 27,
     RS_WINCH = 28,
     RS_IO = 29,
-    RS_PWR	= 30
+    RS_PWR = 30,
```
Comment (doujiang24) [inline resty_signal.c:34]: need to update the doc for these new signal names.

**Pull request merged**